gradle/libs.versions.toml

```diff
@@ -62,6 +62,7 @@ wear = "1.3.0"
 wearComposeFoundation = "1.4.1"
 wearComposeMaterial = "1.4.1"
 wearToolingPreview = "1.0.0"
+junitVersion = "1.2.1"
```

Comment:
Nit - this should be `androidx-test-junit` since this is a version for androidx.test extensions of `junit`, not `junit` itself which there's also a version for defined above.

Reply:
Changed.